orderbook/src/api/filter.rs

```diff
@@ -17,6 +18,11 @@ fn extract_user_order() -> impl Filter<Extract = (UserOrder,), Error = warp::Rej
     warp::body::content_length_limit(MAX_JSON_BODY_PAYLOAD).and(warp::body::json())
 }
 
+fn extract_sell_token() -> impl Filter<Extract = (H160,), Error = warp::Rejection> + Clone {
+    // (rejecting huge payloads)...
+    warp::body::content_length_limit(MAX_JSON_BODY_PAYLOAD).and(warp::body::json())
```

Comment:
This extracts the sell token from the body but I think it is supposed to be a url parameter.

Reply:
Maybe we should update the corresponding openapi change in this PR too.

Reply:
This was actually on purpose. We wanted to have it that way, in order to add easily more parameters later on. This might be needed, if we want to subsidize particular trades, eg. above certain sellAmounts, between certain tokens, only fill/Kill orders etc.

Reply:
I must have missed this specification detail in the API proposal. It makes using the API e.g. in a normal browser harder (to test something locally you'd always have to use curl). Maybe we could specify the sell token in the URL to get the default fee and have any extra parameters that we might want to consider later in the body?

Also I don't see how adding parameters to the body is simpler than to the URL? They could also be optional in the URL in order to be backwards compatible.